decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
I'm tempted to change this to `skip_authorize_resource`. Or maybe even inheriting directly from `::ApplicationController`...

We should really keep this controller _as_ light as possible because any errors raised here will trigger and infinite error rendering loop. Rails detects this loop very early and renders a default error page, but we should avoid that from happening.

By the way, I took the ideas for this patch from https://mattbrictson.com/dynamic-rails-error-pages.